### PR TITLE
chore: deploy packed facets to immutable diamond

### PR DESCRIPTION
**Pull request by ezynda3**

### CODE REVIEW

File: deployments/mainnet.diamond.immutable.json [hunk 1]
```diff
@@ -37,8 +37,8 @@
         "Name": "AmarokFacet",
         "Version": ""
       },
-      "0xA1Edc2eD671Dfa77eD2dCD2ee012F82e4807A75a": {
-        "Name": "CBridgeFacet",
+      "0x3e08304D9e3feb632D2Ee027634785d51fBF9870": {
+        "Name": "CBridgeFacetPacked",
         "Version": ""
```
Comment (satoshi-lifi) [inline deployments/mainnet.diamond.immutable.json:42]: I think it should specify the version?

File: deployments/arbitrum.diamond.immutable.json [hunk 2]
```diff
@@ -49,9 +49,9 @@
         "Name": "HopFacetOptimized",
         "Version": "2.0.0"
       },
-      "0xA57862295BdF8A680fda893B9bd3b935236d18bA": {
+      "0x58FBa880b973edB5E4f6Aa4e2bD3b15c865b9B18": {
         "Name": "HopFacetPacked",
-        "Version": "1.0.1"
+        "Version": ""
```
Comment (satoshi-lifi) [inline deployments/arbitrum.diamond.immutable.json:54]: We don't need to specify the version here?

File: deployments/_deployments_log_file.json [hunk 3]
```diff
@@ -2010,6 +2036,58 @@
...
+    },
+    "arbitrum": {
+      "production": {
+        "1.0.1": [
+          {
+            "ADDRESS": "0x3e08304D9e3feb632D2Ee027634785d51fBF9870",
+            "OPTIMIZER_RUNS": "1000000",
+            "TIMESTAMP": "2023-05-15 20:06:52",
+            "CONSTRUCTOR_ARGS": "0x0000000000000000000000001619de6b6b20ed217a58d00f37b9d47c7663feca00000000000000000000000011f11121df7256c40339393b0fb045321022ce44",
+            "VERIFIED": "false"
```
Comment (0xDEnYO) [inline deployments/_deployments_log_file.json:2061]: Would be good to verify contracts before merging

File: deployments/arbitrum.diamond.immutable.json (hunk 2 above)
Comment (0xDEnYO) [reply deployments/arbitrum.diamond.immutable.json:54]: agreed. This should not be happening. Please try to re-run it.

**Review by 0xDEnYO — CHANGES_REQUESTED** [review]

File: deployments/bsc.diamond.immutable.json [hunk 4]
```diff
@@ -37,9 +37,9 @@
         "Name": "AxelarFacet",
         "Version": ""
       },
-      "0xA1Edc2eD671Dfa77eD2dCD2ee012F82e4807A75a": {
-        "Name": "CBridgeFacet",
-        "Version": ""
+      "0x3e08304D9e3feb632D2Ee027634785d51fBF9870": {
+        "Name": "CBridgeFacetPacked",
+        "Version": "1.0.1"
```
Comment (maxklenk) [inline deployments/bsc.diamond.immutable.json:42]: `CBridgeFacet` and `CBridgeFacetPacked` are two different facets, both should be registered.

Comment (maxklenk) [reply deployments/bsc.diamond.immutable.json:42]: Same applies to other chains.

**Pull request merged**